File: API/Models/CpuLog.cs
namespace API.Models;

public class CpuLog : LogBase
{
	public double Usage { get; }
	public int NumberOfTasks { get; }

	public CpuLog(int serverId, DateTime date, int interval, double usage, int numberOfTasks) : base(serverId, date, interval)
	{
		Usage = usage;
		NumberOfTasks = numberOfTasks;
	}
}

File: website/src/models/CpuLog.ts
export class CpuLog {
  date: string;
  usage: number;
  numberOfTasks: number;

  constructor(date: string, usage: number, numberOfTasks: number) {
    this.date = date;
    this.usage = usage;
    this.numberOfTasks = numberOfTasks;
  }
}
